installer/install.py

```diff
@@ -205,7 +206,27 @@ def traverse_ino(path, ino):
     traverse_ino("", vol.root)
     return packs
 
+def is_vfat(device):
+    try:
+        blkid_output = subprocess.check_output(['blkid', device], stderr=subprocess.DEVNULL)
+    except subprocess.CalledProcessError:
+        report_failure("is_vfat blkid {device} failed.")
+
+    partition_type = blkid_output.decode('ascii').strip()
+    print( partition_type )
+    return re.search(r"vfat|FAT32|FAT16", partition_type) is not None
+
+#def format_sdcard():
+
+if os.path.exists("/dev/mmcblk2p2"):
```

Comment:
I think this actually will not solve the problem where the SD Card is formatted with an extended partition table -- in that case, we get mmcblk2p1 and mmcblk2p5 (!).

I wonder if the thing we actually mean to do is something like:
```python
with open("/proc/mounts", "r") as f:
    mounts = f.read().strip().split('\n') 
found_sdcard = False
for m in mounts:
    dev, pt, fs = m.split(' ')[:3]
    if pt == "/mnt/sdcard":
        if dev != "/dev/mmcblk2p1":
            fail...
        if fs != "vfat":
            fail...
        found_sdcard = True
if not found_sdcard:
    fail...
```

... which tests for precisely what we mean: "the SD card that we are about to write to is mounted in precisely the way that we will mount it on the way up when we boot X1Plus"?